api/server/collections.ts

```diff
@@ -0,0 +1,37 @@
+import { MongoObservable } from 'meteor-rxjs';
```

Comment:
Well, so far all the methods were in one file, all the publications were in one file, so why not putting all collections and models in one file? As for now the server is pretty simple compared to the client so I find it reasonable, what's for sure, I didn't see any purpose into splitting the entities into `whatsapp` and `ufs`.